native/golang/cmds/uss.go

```diff
@@ -239,12 +239,23 @@ func HandleWriteFileRequest(conn *utils.StdioConn, params []byte) (result any, e
 		}
 	}
 
+	tokens := strings.Fields(strings.TrimSpace(string(out)))
+	etag := ""
+	created := false
+	if len(tokens) > 0 {
+		etag = tokens[0]
+	}
+	if len(tokens) > 1 && tokens[1] == "true" {
+		created = true
+	}
```

Comment:
Could we use a more standardized data format instead of spaces to separate data? For example, we could print the output in YAML format from `zowex` and then parse it as YAML in the Go layer using `utils.YamlToMap`.